build.gradle

```diff
@@ -18,7 +18,7 @@ apply plugin: 'nexus'
 apply plugin: 'codenarc'
 
 group = 'com.github.ben-manes'
-version = '0.17.0'
+version = '0.18.0'
```

Comment:
We don't need to make that change yet, do we?

Reply:
No and i will revert this change. my reasoning was that i like to increment the version number before development.